test/arbitrum/JPEGzMainnetOracle.t.sol

```diff
@@ -0,0 +1,505 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.5;
+
+import "forge-std/Test.sol";
+import "../../contracts/ETHVaultHandler.sol";
+import "../../contracts/Orchestrator.sol";
+import "../../contracts/TCAP.sol";
+import "../../contracts/oracles/ChainlinkOracle.sol";
+import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
+
+contract JPEGzMainnetTest is Test {
+  // events
+  event NewMinimumTCAP(address indexed _owner, uint256 _minimumTCAP);
+  event NewMintFee(address indexed _owner, uint256 _mintFee);
+  event NewBurnFee(address indexed _owner, uint256 _burnFee);
+
+  // Setup
+  ETHVaultHandler ethVault;
+  Orchestrator orchestrator = new Orchestrator(address(this));
+  TCAP jpegz = new TCAP("JPEGz Token", "JPEGz", 0, (orchestrator));
+  AggregatorV3Interface jpegzAggregator =
+    AggregatorV3Interface(0x8D0e319eBAA8DF32e088e469062F85abF2eBe599);
+  AggregatorV3Interface ethAggregator =
+    AggregatorV3Interface(0x639Fe6ab55C921f74e7fac1ee960C0B6293ba612);
+  ChainlinkOracle jpegzOracle =
+    new ChainlinkOracle(address(jpegzAggregator), address(this));
+  ChainlinkOracle ethOracle =
+    new ChainlinkOracle(address(ethAggregator), address(this));
+  address weth = 0x82aF49447D8a07e3bd95BD0d56f35241523fBab1;
+  address user = address(0x1);
```

Comment:
Please change this address as this is a precompile address. 
I use something like address(0x51);